CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.0-28
+
+- `MapboxVision` v0.10.0
+- Added detection of construction cones
+- General bugfixes
```

Comment:
Should we describe specific issues that were fixed?

Reply:
I think no, they are related to the life cycle of Vision classes, not user actions, so it would be difficult to explain them without naming specific classes

Reply:
I'm ok to leave it as it is. Probably it would be more convenient to give more details anyway, but it's better to discuss it later, no blockers to merge now.